crates/bevy_reflect/Cargo.toml

```diff
@@ -10,12 +10,13 @@ keywords = ["bevy"]
 readme = "README.md"
 
 [features]
-default = []
+default = ["smallvec"]
```

Comment:
Having just this feature in default feels wrong :thinking:

Reply:
This makes it a non-breaking change because previously it wasn't optional.